lib/directory.coffee

```diff
@@ -159,8 +159,11 @@ class Directory
       names = fs.readdirSync(@path)
     catch error
       names = []
-    names.sort (name1, name2) -> name1.toLowerCase().localeCompare(name2.toLowerCase())
-    
+    unless atom.config.get('tree-view.posixSort')
+      names.sort(naturalCompare);
```

Comment:
the struggles when switching between javascript and coffeescript all day long :)